internal/testgen/gen_negative_positive.go

```diff
@@ -19,28 +20,58 @@ func (g NegativePositiveTestsGenerator) TemplateData() any {
 		report  = checker + ": use %s.%s"
 	)
 
+	var invalidAssertions []Assertion
+
+	for _, zero := range []string{
+		"0",
+		"int(0)", "int8(0)", "int16(0)", "int32(0)", "int64(0)",
+		"uint(0)", "uint8(0)", "uint16(0)", "uint32(0)", "uint64(0)",
```

Comment:
Negative/Positive looks useless for unsigned values.

they are always Positive and never Negative

looks like a case for `incorrect-assert` checker (not implemented yet)

Reply:
Indeed, it's obvious when you spot it.

Unsigned int cannot be negative.

But, unsigned int can be positive, so they leads to compare they are not zero/empty.

To be defined, because then it becomes closed to zero (more than empty)